docs/ref/django-admin.txt

```diff
@@ -418,6 +418,15 @@ models that may have been removed since the fixture was originally generated.
 The :djadminopt:`--app` option can be used to specify a single app to look
 for fixtures in rather than looking through all apps.
 
+The :djadminopt:`--exclude` option may be provided to load only specific
```

Comment:
IMO modified description from similar option of `dumpdata` would fit better: 
> The :djadminopt:`--exclude` option may be provided to prevent specific
applications or models (specified as in the form of ``app_label.ModelName``)
from being loaded.